core/test/lib/libledger-test/FakeUrlConnection.cpp

```diff
@@ -0,0 +1,39 @@
+#include "FakeUrlConnection.hpp"
+#include "api/HttpReadBodyResult.hpp"
+#include "utils/optional.hpp"
+#include "api/Error.hpp"
+
+namespace ledger {
+namespace core {

```

Comment:
That namespace indentation is not the same one as you did with other files, where you increment the indent level. You can stick to one style, please? :)

Reply:
Done